File: julia/1. multiples of 3 or 5.jl
# Multiples of 3 or 5
# https://projecteuler.net/problem=1

multiples = []

for i in 1:999
    if i % 3 == 0 || i % 5 == 0
        push!(multiples, i)
    end
end

println(sum(multiples))

File: javascript/1. multiples of 3 or 5.js
// Multiples of 3 or 5
// https://projecteuler.net/problem=1

function multiplesOf3and5(number) {
  const multiples = [];
  for (let i = 3; i < number; i++) {
    if (i % 3 == 0 || i % 5 == 0) {
      multiples.push(i);
    }
  }
  let sum = multiples.reduce((acc, value) => acc + value);
  return sum;
}

console.log(multiplesOf3and5(1000));
